docs/tutorial/05-non-fungible-tokens-1.md

```diff
@@ -51,10 +57,10 @@ Possible examples of NFTs include:
 CryptoKitties, Top Shot Moments, and tickets to a really fun concert.
 
 Instead of being represented in a central ledger, like in most smart contract languages,
-Cadence represents each NFT as a [resource object](../language/composite-types)
+Cadence represents each NFT as a [resource object](../language/composite-types.mdx)
```

Comment:
Is https://cadence-lang-org-git-josh-c1-tutorials-onflow.vercel.app/docs/language/resources a better spot to link to?

Reply:
No, I think it is better to link to the language reference